### DISCUSSION TITLE
Why is there no official docker image?

### DISCUSSION THREAD

**Discussion by Aragur** [pnpm/pnpm#4944] [Q&A] [answered]
So I have found this repository: https://github.com/pnpm/docker-pnpm  
But the Docker Hub link is dead.  
I'm currently trying to figure out how to use pnpm in my drone pipeline.  
I'm using the standard `node:lts-alpine` image.  
This script runs successfully `npm i -g npm`  but after that i cannot run any pnpm command (`/bin/sh: pnpm: not found`).  
What's the best practice working with pnpm on a ci? Currently I'm using npm but that doesn't feel right, or is this the way to go?

**Comment by CobyPear** [comment, marked as answer]
In my experience, it's fine to install pnpm with npm in CI, or use [the `curl` option](https://pnpm.io/installation#using-a-standalone-script). Looks like all of the CI recipes use the curl option https://pnpm.io/continuous-integration

**Reply by Aragur** [reply]
Yes I saw that.  
But on the official node image there is no curl available.

**Reply by CobyPear** [reply]
Oh that's interesting. I have had success with [the `corepack` option](https://pnpm.io/installation#using-corepack) in CI, but it's only available on node v16.13+

**Reply by Aragur** [reply]
Thanks a lot. Never heard of corepack before 🤯 
Using `corepack pnpm@7.3.0 install` works like a charm.

**Reply by michaelact** [reply]
You can also try `npm install -g pnpm@<version>`.

**Comment by sep2** [comment]
As corepack will be removed from node soon, do we have new options here?
https://github.com/nodejs/package-maintenance/pull/606

**Reply by TehBrian** [reply]
A pnpm Docker image would be nice.

**Comment by iTrooz** [comment]
There is one now: ghcr.io/pnpm/pnpm

**Reply by TehBrian** [reply]
This is nice. Thank you for sharing :-)

**Comment by devzom** [comment]
### 🔥  Official PNPM Docker image available 

Finally we have an access to tuned PNPM image to be used in Dockerfiles 🔥 🔥 🔥 

https://pnpm.io/docker#installing-nodejs
